web/modules/custom/dpl_patron_menu/src/Plugin/Block/PatronMenuBlock.php

```diff
@@ -55,15 +64,18 @@ public function __construct(array $configuration, $plugin_id, $plugin_definition
    * @param int $plugin_definition
    *   The plugin implementation definition.
    *
-   * @return \Drupal\dpl_loans\Plugin\Block\LoanListBlock|static
-   *   Loan list block.
+   * @return \Drupal\dpl_patron_menu\Plugin\Block\PatronMenuBlock|static
+   *   Patron menu block.
    */
   public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
     return new static(
       $configuration,
       $plugin_id,
       $plugin_definition,
       $container->get('config.factory'),
+      $container->get('dpl_library_agency.branch_settings'),
+      $container->get('dpl_library_agency.branch.repository'),
+      \Drupal::service('dpl_patron_menu.settings')
```

Comment:
It should be possible to get it from the container. Isn't it possible to get it like:
`$container->get('dpl_library_agency.branch.repository')`?

Reply:
I think it would be, but this follows the pattern used in all the other modules.